DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/ViewDelegate/ConfirmRemoveFromCourse.cshtml

```diff
@@ -14,8 +14,9 @@
     @if (errorHasOccurred) {
       <vc:error-summary order-of-property-names="@(new[] { nameof(Model.Confirm) })" />
     }
-    <h1 class="nhsuk-heading-xl nhsuk-u-margin-bottom-3 word-break">Remove delegate enrolment from @Model.CourseName</h1>
+    <h1 class="nhsuk-heading-xl nhsuk-u-margin-bottom-3 word-break">Remove enrolment</h1>

```

Comment:
Think this page is listed in the automated accessibility tests, and will need that test parameter changing to match this new h1.